api-admin/src/main/java/bio/terra/pearl/api/admin/service/enrollee/ParticipantTaskExtService.java

```diff
@@ -93,4 +92,44 @@ public List<ParticipantTask> updateTasks(
             studyEnv.getId(), updateDto, new ResponsibleEntity(operator));
     return updatedTasks;
   }
+
+  public ParticipantTaskService.ParticipantTaskTaskListDto getByStudyEnvironment(
```

Comment:
nit: should we use auth annotations for these?

Reply:
I thought about it- but I decided not to do it as a part of this PR just because it was already getting a bit large and I didn't want to mix auth changes in with a moderately large refactor